config/v1/types_node.go

```diff
@@ -48,7 +48,12 @@ type NodeSpec struct {
 	WorkerLatencyProfile WorkerLatencyProfileType `json:"workerLatencyProfile,omitempty"`
 }
 
-type NodeStatus struct{}
+type NodeStatus struct {
+	// conditions represent the observations of the current state.
```

Comment:
should the go doc have more information like that in the description in the crds below?

Reply:
Added a generic info about the conditions field.